include/medium/elastic_anisotropic2d/elastic_anisotropic2d_properties.hpp

```diff
@@ -1,146 +1,16 @@
 #pragma once
 
 #include "enumerations/specfem_enums.hpp"
+#include "medium/properties.hpp"
 #include "point/properties.hpp"
-#include "properties.hpp"
 #include "specfem_setup.hpp"
 #include <exception>
 #include <iostream>
 #include <ostream>
 #include <sstream>
 
 namespace specfem {
-namespace material {
```

Comment:
We might wanna discuss if properties should be in a separate file. I like it being in its separate file but it kinda goes against "need to change only 1 file" when implementing new material systems

Reply:
We changed it to "change only one directory!".